src/templates/doe_workflow_attic_insulation.mdx

```diff
@@ -0,0 +1,188 @@
+<Tabs>
+    <Tab eventKey="project" title="Project">
+        The data for this tab should be know prior to the initial site visit.
+
+        <ProjectInfoInputs {...props} />   
+
+    </Tab>
+    <Tab eventKey="installation" title="Installation">
+        
+        <DateInput label="Installation Date" path="install_date" />
+        
+        ## Building
+        <PhotoInput id="building_number_photo" label="Building Number – Photo">
+         Provide a photo of the building that shows the street number.
+        </PhotoInput>
+
+        Skip photo if feature is N/A to house.
+        
+        <PhotoInput id="insulation_measuring_rulers " label="Insulation Measuring Rulers  – Photo ">
+            Overview of insulation measuring rulers, one installed every 300 sq. ft. visible 
+            from attic entrance (photo taken from attic entrance).     
+        </PhotoInput>
+
+        <PhotoInput id="rvalue" label="R-value Per Inch  – Photo ">
+        Photo of insulation bag R-value per inch table    
+        </PhotoInput>
+
+        <PhotoInput id="skylight_shaft" label="Skylight Shaft – Photo ">
+        Photot of skylight shaft insulation detail
+        </PhotoInput>
+
+        <PhotoInput id="knee_wall" label="Knee Wall – Photo ">
+            Photo of knee wall insulation detail     
+        </PhotoInput>
+
+        <PhotoInput id="stairwell_wall" label="Stairwell Wall – Photo ">
+        Photo of stairwell wall insulation detail     
+        </PhotoInput>
+
+        <PhotoInput id="storage_platform" label="Storage Platform – Photo ">
+        Photo of storage platform insulation detail    
+        </PhotoInput>
+
+        <PhotoInput id="wide_angle_photo" label="Wide Angle – Photo ">
+        Wide angle photo showing insulation uniformity with rulers visible
+        </PhotoInput>
+
+         <PhotoInput id="atticdoor_hatch" label="Attic Door/Hatch - Photo ">
+        Photo of insulation on attic door/hatch
+        </PhotoInput>
+    
+    </Tab>
+    <Tab eventKey="postassessment" title="Post Assessment">
```

Comment:
Just for consistency, the event key should probably be "post_assessment".

Reply:
done